jquery_dom_selector_practice/js/exercise.js

```diff
@@ -13,22 +13,23 @@
  * We've started you off, replace the sec
  */
  
+ $("#button1").click(question1);
 	function question1(){
-		$("replaceMe").css("replaceMe", "replaceMe");
+		$('p').css("color", "blue");
```

Comment:
this is definitely not a big deal but developers can get real antsy about being consistent with single quotes and double quotes. So try to be consistent in your usage